samples/system-test/detect.test.js

```diff
@@ -193,7 +193,6 @@ test(`should detect crop hints in a remote file`, async t => {
 test(`should detect similar web images in a local file`, async t => {
   const output = await tools.runAsync(`${cmd} web ${files[5].localPath}`, cwd);
   t.true(output.includes('Full matches found:'));
-  t.true(output.includes('URL: https://cloud.google.com/vision/docs/images/'));
```

Comment:
Hrm, this was breaking for me during tests runs, but I just got this as a result again when I ran this command... I will add this back presuming the result is still consistently returned

Reply:
Confirmed, this is not consistently returned

    $ node detect.js web resources/faulkner.jpg

That's why this was removed 😄

Reply:
The joys of system testing in ML.